synapse/rest/client/room.py

```diff
@@ -1284,25 +1284,22 @@ class TimestampLookupRestServlet(RestServlet):
     `dir` can be `f` or `b` to indicate forwards and backwards in time from the
     given timestamp.
 
-    GET /_matrix/client/unstable/org.matrix.msc3030/rooms/<roomID>/timestamp_to_event?ts=<timestamp>&dir=<direction>
+    GET /_matrix/client/v1/rooms/<roomID>/timestamp_to_event?ts=<timestamp>&dir=<direction>
     {
         "event_id": ...
     }
     """
 
-    PATTERNS = (
-        re.compile(
-            "^/_matrix/client/unstable/org.matrix.msc3030"
-            "/rooms/(?P<room_id>[^/]*)/timestamp_to_event$"
-        ),
-    )
+    PATTERNS = (re.compile("^/_matrix/client/v1/rooms/(?P<room_id>[^/]*)/timestamp_to_event$"),)
```

Comment:
I just matched what `/threads` was doing since `client_patterns(...)` seems to be doing weird stuff with `api/v1` and `unstable` by default. Not sure why those are things for normal endpoints.

https://github.com/matrix-org/synapse/blob/a84744fba0b123eef31a179648177e9d4aacc693/synapse/rest/client/relations.py#L85

Reply:
Moved to https://github.com/matrix-org/synapse/pull/14471#discussion_r1024523025 since this got outdated